spacy_llm/models/bedrock/registry.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Callable, Dict, Iterable, List
+
+from confection import SimpleFrozenDict
+
+from ...registry import registry
+from .model import Bedrock, Models
+
+_DEFAULT_RETRIES: int = 5
+_DEFAULT_TEMPERATURE: float = 0.0
+_DEFAULT_MAX_TOKEN_COUNT: int = 512
+_DEFAULT_TOP_P: int = 1
+_DEFAULT_STOP_SEQUENCES: List[str] = []
+
+
+@registry.llm_models("spacy.Bedrock.Titan.Express.v1")
```

Comment:
I think Titan Express and Titan Lite are quite similar in their usage, so let's merge this to one registry entry.